src/common/utils.js

```diff
@@ -100,10 +103,32 @@ export function copyToClipboard(data) {
   textField.remove();
 };
 
+/* Returns date set to local timezone and in format of locale */
+export function formatLocalDate(date, isUTC, timezone) {
+  if(!timezone) {
+    timezone = moment.tz.guess();
+  }
+  if (isUTC) {
+    date = moment.utc(date);
+  }
+  var localDate = moment.tz(date, timezone);
+  return localDate.format("l") + " " + localDate.format("LTS");
```

Comment:
"l" = 9/4/1986, "LTS" = 8:30:25 PM, (both are locale-specific). Moment does not have a single locale string for our desired date format.